pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -23,6 +23,7 @@ const (
 	// -480 for Asia/Shanghai(+8)
 	ParamTimeZoneOffset = "timeZoneOffset"
 	ParamSchedule       = "schedule"
+	CleverWorkflowName  = "clever.caicloud.io/workflow-name"
```

Comment:
Please define const in `pkg/workflow/common/const.go` follow `WorkflowRunAnnotationName`, like 

```
WorkflowRunAnnotationName = "cyclone.io/workflowrun"
WorkflowAnnotationName = "cyclone.io/workflow"
```

Don't use `clever`, `caicloud` terms as we are creating a open source project.

Reply:
OK, I will move it and use `cyclone.io/workflow`